### DISCUSSION TITLE
Can't override tooltip Zindex property, when the tooltip is inside a modal

### DISCUSSION THREAD

**Discussion by shiranZe** [chakra-ui/chakra-ui#5238] [Q&A]
Hi,

I'm using Chakra-Ui `Tooltip` component inside a modal (the modal is not a Chakra-Ui component).
The modal has a `z-index=9999` value.
I gave the tooltip a higher `zindex` value, but I can still see that the original zindex wasn't override.

This is my theme file:

```
const baseStyle = {
  px: 2,
  py: 1,
  bg: 'black',
  color: 'white',
  borderRadius: 'lg',
  maxW: '228px',
  zIndex: 'tooltip',
  textAlign: 'center',
};

export default {
  baseStyle,
};  
```

and here is the usage:

```
<Tooltip
              isOpen={options < 2}
              placement="right-start"
              label={tootltipLabel}
              zIndex={99999}
            >
              <WizardSelectField
                placeholder={label}
              />
</Tooltip>
```

A snapshot from devtools: (You can see that the zIndex value is still `chakra-tooltip`

![image](https://user-images.githubusercontent.com/40235967/145831465-6b275b7a-1094-47f0-9224-1b3c2b6fa3f8.png)

How can I override the theme?

Thank you!

**Comment by lostgirljourney** [comment]
Hi, I went through a similar situation where I needed my tooltip to be behind the nav that I made but it was still popping up over my nav. In this case, you can use Chakra UI Custom Theme. Here's the documentation link: [tooltip/theming](https://chakra-ui.com/docs/components/tooltip/theming) & also, [docs/styled-system/theme#z-index-values](https://chakra-ui.com/docs/styled-system/theme#z-index-values).

This is what I did from the documentation as the situation was my drawer should be placed above my nav & tooltip be below them, so theming was a better option.
- `component/tooltip.ts`
```ts
import { defineStyleConfig } from '@chakra-ui/react';

// define the base component styles
const baseStyle = {
	zIndex: 999
};

// export the component theme
export const tooltipTheme = defineStyleConfig({
	baseStyle
});
```

- `theme/index.ts`
```ts
import { extendTheme } from '@chakra-ui/react';
import { tooltipTheme } from './component/Tooltip';

const theme = extendTheme({
	components: {
		Tooltip: tooltipTheme
	}
});

export default theme;
```

One might understand from here as well: [https://stackoverflow.com/questions/74488941/how-to-override-chakra-ui-modal-component-z-index-value-to-place-modal-component](https://stackoverflow.com/questions/74488941/how-to-override-chakra-ui-modal-component-z-index-value-to-place-modal-component)
> Writing this answer because might in the future, someone needs it as I did.

**Comment by cie** [comment]
```tsx
 <Tooltip
   sx={{ '[data-popper-placement]:has(>&)': { zIndex: 'banner' } }}
```
